Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -758,11 +758,11 @@ - (void)configureForThread:(TSThread *)thread
         [[YapDatabaseViewMappings alloc] initWithGroups:@[ thread.uniqueId ] view:TSMessageDatabaseViewExtensionName];
     [self.uiDatabaseConnection readWithBlock:^(YapDatabaseReadTransaction *transaction) {
       [self.messageMappings updateWithTransaction:transaction];
-      self.page = 0;
-      [self updateRangeOptionsForPage:self.page];
-      [self.collectionView reloadData];
     }];
+    self.page = 0;
```

Comment:
Hmm... this seems like a reasonable way to maybe avoid some kind of layout race condition.  👍 

Did you observe a change, or is this a hunch?